client/packages/requisitions/src/RequestRequisition/DetailView/AppBarButtons/AppBarButtons.tsx

```diff
@@ -59,6 +59,7 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
         <ReportSelector
           context={ReportContext.Requisition}
           onPrint={printReport}
+          queryParams={{ filterBy: { subContext: { equalAnyOrNull: [] } } }}
```

Comment:
Would have preferred `notEqualTo: "R&R"`, but this only returns reports where subcontext is not null as well... and its my understanding that Requisition reports don't use any subcontexts? Any risk in adding this filter now??

Reply:
this has the advantage of also excluding the next sub-context-specific report as well as the 'R&R' one. Using the notEqualTo would require keeping up to date if we did this again 🤷 

shame though - I thought that `notEqualTo` would work in this case. It needs to be a notEqualToOrNull I guess

Reply:
this line might need a comment explaining why it was necessary. If I was coming in cold, I'd probably say we could remove that filter!